src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -78,6 +79,7 @@ def load_arguments(self, _):
         c.argument('disk_name', existing_disk_name, completer=get_resource_name_completion_list('Microsoft.Compute/disks'))
         c.argument('name', arg_type=name_arg_type)
         c.argument('sku', arg_type=disk_sku, help='Underlying storage SKU')
+        c.argument('os_type', arg_type=get_enum_type(OperatingSystemTypes), help='The Operating System type of the Disk.')
```

Comment:
please make sure we don't need api-version scope

Reply:
[`OperatingSystemTypes`](https://docs.microsoft.com/en-us/python/api/azure-mgmt-compute/azure.mgmt.compute.v2015_06_15.models.compute_management_client_enums.operatingsystemtypes?view=azure-python) is available in the oldest API version (2015-06-15). So we shouldn't need a min api version scope right?